src/main/resources/reference.conf

```diff
@@ -243,15 +243,15 @@ divolte {
     // the Kafka producer.
     // See: http://kafka.apache.org/documentation.html#producerconfigs
     producer = {
-      metadata.broker.list = ["localhost:9092"]
-      metadata.broker.list = ${?DIVOLTE_KAFKA_BROKER_LIST}
-
+      bootstrap.servers = ["localhost:9092"]
+      bootstrap.servers = ${?DIVOLTE_KAFKA_BROKER_LIST}
       client.id = divolte.collector
       client.id = ${?DIVOLTE_KAFKA_CLIENT_ID}
 
-      request.required.acks = 0
-      message.send.max.retries = 5
-      retry.backoff.ms = 200
+      acks = 1
+      retries = 0
+      compression.type = lz4
+      max.in.flight.requests.per.connection = 1
```

Comment:
Divolte's code already implies the possibility of re-ordering in the case of retry logic, and this isn't configurable by the user. Setting to 1 in flight just (severely) reduces efficacy of compression as well as harms throughput if there is any network latency. Of course, having two levels of retry logic *will* cause some very confusing results no doubt.

Yes, you should drop the event instead of retrying, because there buffer is already full and retries have already been attempted. As per the logic in the ProcessingPool.enqueue(), when the buffer is full, you don't retry, and the only indication of a failure is a warning message with no context information about which event was dropped. So allowing a BufferExhaustedException to be trapped & logged by that code creates the same effect with less complexity and ambiguity.